tests/acceptance/blocks/facetscomponent.js

```diff
@@ -0,0 +1,48 @@
+import FilterBoxComponentBlock from './filterboxcomponent';
+
+/**
+ * This class models user interactions with the {@link FacetsComponent}.
+ */
+export default class FacetsComponentBlock {
+  constructor () {
+    this._filterBox = new FilterBoxComponentBlock('.js-yxt-Facets');
+  }
+
+  /**
+   * Return the child filter box component's block.
+   */
+  async getFilterBox () {
+    return this._filterBox;
+  }
+
+  /**
+   * Toggle the given option for the given filter.
+   */
+  async toggleOption (filterIndex, optionIndex) {
```

Comment:
I think we could make this more user-friendly by allowing someone to select a filter by name and an option by name.